web_advanced_filter/models/ir_filters.py

```diff
@@ -0,0 +1,215 @@
+# Copyright 2014-2020 Therp BV <https://therp.nl>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+import logging
+from odoo import _, api, fields, models
+from odoo.tools.safe_eval import safe_eval
+from odoo.osv import expression
+
+
+_logger = logging.getLogger(__name__)
+
+
+class IrFilters(models.Model):
+    _inherit = 'ir.filters'
+    _evaluate_before_negate = ['one2many', 'many2many']
+
+    is_frozen = fields.Boolean('Frozen', compute='_compute_is_frozen')
+    union_filter_ids = fields.Many2many(
+        'ir.filters', 'ir_filters_union_rel', 'left_filter_id',
+        'right_filter_id', 'Add result of filters',
+        domain=['|', ('active', '=', False), ('active', '=', True)],
```

Comment:
This works too:
```suggestion
        context={'active_test': False},
```
(idem for `complement_filter_ids`)
The field context isn't inherited by the returned recordset, so I think it's exactly equivalent.